app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
         <activity
             android:name=".ActivityMain"
             android:configChanges="orientation|screenSize"
-            android:label="Main"
+            android:label="NetGuard"
```

Comment:
This suggestion is okay, but it needs to be a string resource reference to the application name.